website/content/docs/secrets/databases/index.mdx

```diff
@@ -57,15 +57,16 @@ management tool.
    By default, the secrets engine will enable at the name of the engine. To
    enable the secrets engine at a different path, use the `-path` argument.
 
-1. Configure Vault with the proper plugin and connection information:
+2. Configure Vault with the proper plugin and connection information:
```

Comment:
I believe that our markdown uses the same syntax as described [here for numbered lists](https://www.markdownguide.org/basic-syntax/#ordered-lists). Basically you can start with `1.` and all subsequent list items can also use `1.`. The generator will increment the numbers. This is useful for long lists or refactoring/changing the order of list items.

You can see the final output [here](https://www.vaultproject.io/docs/secrets/databases#setup) that even though the raw text file has `1.`'s it has incremented them in the final output.

Reply:
I did notice this and it makes sense. I changed it to match the other Consul doc, but now I noticed most of the docs do this instead. I'll revert the numbering and instead update the other file to fit with the rest.